### ISSUE TITLE
Check CI failures

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#4964]
Looks like there's some problem with node versions:

```
Run sudo yarn global add get-firefox
  sudo yarn global add get-firefox
  get-firefox --platform linux --branch devedition --extract --target $HOME
  echo "$HOME/firefox" >> $GITHUB_PATH
  shell: /usr/bin/bash -e {0}
yarn global v1.22.22
[1/4] Resolving packages...
[2/4] Fetching packages...
error taskcluster-client@60.4.2: The engine "node" is incompatible with this module. Expected version "18.19.1". Got "18.20.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/global for documentation about this command.
Error: Process completed with exit code 1.
```

**Comment by bovine3dom** [comment]
problem is `get-firefox`, https://github.com/freaktechnik/get-firefox

the latest release, v6, uses an old version of taskcluster that needs an older version of node.

we can use master `yarn add freaktechnik/get-firefox` but 1) bad idea and 2) that uses a newer version of taskcluster which needs a newer version of node.

more brainpower required

:cry:

**Comment by Dranaxel** [comment]
Hi, can you elaborate why it's not possible to use master version of get-firefox or upgrade node ? If I read correctly there is only one minor difference, it *should* not be a big issue

**Comment by bovine3dom** [comment]
The master branch will be unstable as whatever code gets committed will go into it. So we would get more spurious CI failures.

Upgrading node: maybe it's possible but I don't want to have to upgrade node every time just to use get-firefox.

snap: I don't think you can easily side load extensions so we wouldn't be able to run our tests.

apt-get: dunno, maybe it was so we could install specific versions.

Anyway, mozdownload seems to work, the CI problems are somewhere else now. See #4990

**Comment by Dranaxel** [comment]
According to what I see there is a timeout reached during the await of an async 

func test("`:rssexec` works", async () => {
          |     ^
      127 |         const driver = await getDriver()
      128 |         try {
      129 |             await sendKeys(driver, ":set rsscmd js "

**Comment by bovine3dom** [comment]
Yeah, that's the symptom. The question is why that is happening :)

**Comment by bovine3dom** [comment]
Fixed in #5057
